app/code/community/Algolia/Algoliasearch/Helper/Data.php

```diff
@@ -437,6 +437,46 @@ public function rebuildStoreCategoryIndexPage($storeId, $collectionDefault, $pag
         }
     }
 
+    /**
+     * Check if product can be index on Algolia
+     *
+     * @param Mage_Catalog_Model_Product $product
+     * @param int     $storeId
+     *
+     * @return bool
+     *
+     */
+    public function canProductBeReindexed(Mage_Catalog_Model_Product $product, $storeId)
```

Comment:
How about moving this method to ProductHelper? It's very Product specific method.

Reply:
We could move that over there! We should consider that as well for M2 feature if we want to keep it the same.